playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: adhoc
+  pre_tasks:
+  - name: Print list of known health checks
+    action: openshift_health_check
+    when: checks is undefined or not checks
+  post_tasks:
+  - name: Run health checks
+    action: openshift_health_check
+    args:
+      checks: '{{ checks | default([]) }}'
```

Comment:
Made it `openshift_checks`, thanks for pointing it out.

There is still room for debate when we compare with disable, because the latter is in singular form: `openshift_disable_check`...